<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DanLeiMTFCalc</class>
 <widget class="QWidget" name="DanLeiMTFCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>611</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>721</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Pre-Process Options</string>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>31</y>
       <width>371</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Auto or Manual</string>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>330</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QRadioButton" name="isManualRB">
         <property name="text">
          <string>Manual Process</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="visualFieldAssistCB">
         <property name="text">
          <string>Visual Field Assist</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="isAutoRB">
         <property name="text">
          <string>Auto Batch Process</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>180</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ROI Width:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>ROI Height:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="ROIWSB"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="ROIHSB"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>301</width>
       <height>271</height>
      </rect>
     </property>
     <property name="title">
      <string>Output Unit</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>79</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="ipOrMmRB">
         <property name="text">
          <string>Ip/mm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="CorPRB">
         <property name="text">
          <string>C/P</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="LWOrPHRB">
         <property name="text">
          <string>LW/PH</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>198</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="pixelSizeLB">
         <property name="text">
          <string>Pixel Size(um):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="pixelSizeDB"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>621</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>500</height>
     </size>
    </property>
    <property name="title">
     <string>Visual Field Selection</string>
    </property>
    <widget class="QCheckBox" name="nw9CB">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>44</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne9CB">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>31</y>
       <width>49</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw8CB">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>73</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne8CB">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>60</y>
       <width>79</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw7CB">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>102</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne7CB">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>89</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw6CB">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>131</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne6CB">
     <property name="geometry">
      <rect>
       <x>469</x>
       <y>118</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw5CB">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>159</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne5CB">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>146</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw4CB">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>188</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne4CB">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>175</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw3CB">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>217</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne3CB">
     <property name="geometry">
      <rect>
       <x>377</x>
       <y>204</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw2CB">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>245</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne2CB">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>232</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="nw1CB">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>274</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="ne1CB">
     <property name="geometry">
      <rect>
       <x>316</x>
       <y>261</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="c0CB">
     <property name="geometry">
      <rect>
       <x>286</x>
       <y>290</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw1CB">
     <property name="geometry">
      <rect>
       <x>256</x>
       <y>318</y>
       <width>53</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se1CB">
     <property name="geometry">
      <rect>
       <x>316</x>
       <y>331</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw2CB">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>347</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se2CB">
     <property name="geometry">
      <rect>
       <x>347</x>
       <y>360</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw3CB">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>376</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se3CB">
     <property name="geometry">
      <rect>
       <x>377</x>
       <y>389</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw4CB">
     <property name="geometry">
      <rect>
       <x>164</x>
       <y>404</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se4CB">
     <property name="geometry">
      <rect>
       <x>408</x>
       <y>417</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw5CB">
     <property name="geometry">
      <rect>
       <x>134</x>
       <y>433</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se5CB">
     <property name="geometry">
      <rect>
       <x>438</x>
       <y>446</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw6CB">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>462</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se6CB">
     <property name="geometry">
      <rect>
       <x>469</x>
       <y>475</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw7CB">
     <property name="geometry">
      <rect>
       <x>73</x>
       <y>490</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se7CB">
     <property name="geometry">
      <rect>
       <x>499</x>
       <y>503</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.7</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw8CB">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>519</y>
       <width>54</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se8CB">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>532</y>
       <width>49</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>0.8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sw9CB">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>548</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="se9CB">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>548</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0.9</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>410</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Import Image(s) and Calculate</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>327</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string> Save as</string>
    </property>
    <widget class="QCheckBox" name="isSaveCheck">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Execl工作(.xlsx)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
